### ISSUE TITLE
primary button focus ring missing for `mytheme`

### ISSUE THREAD

**Issue by bobby-driggs** [primefaces/primevue-sass-theme#54]
Not sure if this is a bug or not.

I just updated to version 3.47.2 (from 3.44.0), and now my primary buttons don't have a focus ring when they are pressed (has the `:focus` state active), but the Secondary, Info, Warning, etc buttons still do.

They have the SCSS like below inside the `theme-base\components\button\_button.scss` file
```
    &:not(:disabled):focus {
        box-shadow: $secondaryButtonFocusShadow;
    }
```

I noticed that the non-primary buttons have rules that target the `:focus` state, but the primary button does not have this rule. It looks like this commit changed it to be `:focus-visible` : https://github.com/primefaces/primevue-sass-theme/commit/08026c029a2fdfd2244d0e82e2a61d4965a07c4c

Flipping around the themes on https://primevue.org/button/ it looks like the Aura themes do not have this focus ring, but the Lara theme does.

It seems like the `mytheme` editable base is maybe broken? Or is the intent to manually set the focus ring for the primary button when using the `mytheme` ?
